autogen/json2class.cpp

```diff
@@ -1483,7 +1528,7 @@ void make_class(
 
    // output: defaults (applicable only to metadata)
    oss << "\n   " << small;
-   oss << "\n   // Relevant defaults";
+   oss << "\n   // Relevant defaults;";
```

Comment:
I'm nitpicking but you don't need a semi-colon here

Reply:
I thought it "read" a bit better, but if you don't like it, I'll take it out. :-)